core/src/main/java/io/grpc/ClientStreamTracer.java

```diff
@@ -46,6 +46,12 @@ public void headersSent() {
   }
```

Comment:
Should headersSent be outboundHeaders? I was going to suggest changing inboundHeaders until I realized it matched the methods in StreamTracer better. That can be done in a later PR.